src/Data/Fix.hs

```diff
@@ -266,18 +270,22 @@ fixDataType = mkDataType "Data.Functor.Foldable.Fix" [fixConstr]
 -- | Least fixed point. Efficient folding.
 newtype Mu f = Mu { unMu :: forall a. (f a -> a) -> a }
 
-instance (Functor f, Eq1 f) => Eq (Mu f) where
+instance (Functor f, Eq (f (Fix f))) => Eq (Mu f) where
     (==) = (==) `on` foldMu Fix
 
-instance (Functor f, Ord1 f) => Ord (Mu f) where
+instance (Functor f, Ord (f (Fix f))) => Ord (Mu f) where
```

Comment:
Now thinking more about it. I'm not happy about this change. It exposes too much of implementation detail to the surface. `Ord1 f` is asking for more, but doesn't tell anyone on the outside that `Fix` is used internally.

I'm not particularly happy with this. So I'm sorry, but we'll need to rethink this change.

Changing `Fix` interface is one option, but I don't like that either, as then `Fix` and `Mu/Nu` parts will look different.